build.gradle

```diff
@@ -3,24 +3,34 @@ plugins {
     id 'org.jetbrains.intellij' version '0.4.16'
 }
 
-group 'uk.co.hadoopathome.intellij.avro'
+group 'uk.co.hadoopathome.intellij.viewer'
 version '1.0.10'
 
 repositories {
     mavenCentral()
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8;
-targetCompatibility = JavaVersion.VERSION_1_8;
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
 
 dependencies {
+    //Avro dependencies
     compile("org.apache.avro:avro:1.9.1") {
         exclude group: 'org.slf4j'
     }
     compile("org.xerial.snappy:snappy-java:1.1.7.3")
     compile("com.google.code.gson:gson:2.8.5")
```

Comment:
These should be under an `//External dependencies` header, probably below Parquet.